.eslintrc.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
   ignorePatterns: ["node_modules/"],
   rules: {
     '@typescript-eslint/no-non-null-assertion': 'off',
-    '@typescript-eslint/triple-slash-reference': 'off'
+    '@typescript-eslint/triple-slash-reference': 'off',
+		"no-tabs": 0,
```

Comment:
1. Bad indentation
2. Why is this in this PR? Why allow tabs?

Reply:
this was a temporary lint-get-off-of-my-back thing that I forgot to remove, done